screens/Recipe_Comp/RecipeCard.js

```diff
@@ -315,38 +303,40 @@ const RecipeCard = ({ recipe, RDA }) => {
                   style={{
                     flexDirection: "row",
                     justifyContent: "center",
-                    alignItems: "center",
-                    width: "75%",
                   }}
                 >
                   <TextInput
                     style={styles.servingInputBox}
                     value={servingSize}
                     placeholder={`${servingSize}`}
-                    placeholderTextColor="#696969"
+                    placeholderTextColor="white"
                     keyboardType={"numeric"}
                     onChangeText={(val) => setServingSize(val)}
                   />
                   <RNPickerSelect
                     selectedValue={servingUnit}
-                    placeholder={{}}
-                    style={pickerStyles}
+                    value={servingUnit}
```

Comment:
There was a placeholder={{}} prop in this picker component. I remember it having some additional functionality, but can't remember it at the moment. I think we should add the placeholder if we need to.